.travis.yml

```diff
@@ -1,14 +1,11 @@
 language: go
 go:
   - 1.5.1
-branches:
-  only:
-    - master
```

Comment:
I'm not 100% sure what this means tbh, but I'm guessing what you said RE PRs against master is correct. Anyway, [here](https://travis-ci.org/deis/builder/builds/122652593#L229) is an example PR build that runs `make ... docker-build`